tests/h/search/index_test.py

```diff
@@ -56,22 +72,66 @@ def test_it_indexes_the_annotations_document_title(self, factories,
         assert get(annotation.id)["document"]["title"] == ["test_document_title"]
 
     def test_it_can_index_an_annotation_with_a_document_with_no_title(self, factories,
+                                                                      make_annotation,
                                                                       index, get):
-        annotation = factories.Annotation.build(
-            id="test_annotation_id",
+        annotation = make_annotation(
             document=factories.Document.build(title=None),
         )
 
         index(annotation)
 
         assert "title" not in get(annotation.id)["document"]
 
+    def test_you_can_filter_annotations_by_authority(self, make_annotation, index, search):
+        annotation = make_annotation(userid="acct:someone@example.com")
+
+        index(annotation)
+
+        response = search.filter("term", authority="example.com").execute()
+        assert SearchResponseWithIDs([annotation.id]) == response
+
+    def test_you_can_filter_annotations_by_creation_time(self, factories, index, search):
+        before = datetime.datetime.now()
+        annotation = factories.Annotation.build(id="test_annotation_id", created=datetime.datetime.now())
+
+        index(annotation)
+
+        response = search.filter("range", created={"gte": before}).execute()
+        assert SearchResponseWithIDs([annotation.id]) == response
```

Comment:
These matchers have the form `assert expected == actual` rather than the usual `assert actual == expected`, is there a reason for that?